package.json

```diff
@@ -87,11 +88,11 @@
     "istanbul-instrumenter-loader": "0.2.0",
     "jasmine-core": "2.4.1",
     "karma": "1.3.0",
-    "karma-chrome-launcher": "0.2.3",
-    "karma-cli": "0.1.2",
+    "karma-chrome-launcher": "2.0.0",
```

Comment:
These are changes that the Angular team did in the docs.
Though we found an issue with Protractor, so I downgraded it to 4.0.8.